ui/src/funding/pool.ts

```diff
@@ -328,3 +331,15 @@ function newSetOfReceivers(current: Receivers, changes: Receivers): Receivers {
     [...merged].filter(([_address, status]) => status != ReceiverStatus.Removed)
   );
 }
+
+// Check whether the given value can be converted to `ethers.BigNumber`.
+// N.B: ethers.BigNumber.isBigNumber doesn't work as expected:
+// https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L285
+export function isValidBigNumber(value: string): boolean {
```

Comment:
It doesn't seem ideal to have this here but I don't find a better place, given we don't have a general `util` module and that this function is only used within the scope of the funding pool.